opds_import.py

```diff
@@ -596,7 +607,8 @@ def _data_detail_for_feedparser_entry(cls, entry, metadata_data_source):
             if circulation_data_source_name:
                 _db = Session.object_session(metadata_data_source)
                 circulation_data_source = DataSource.lookup(
-                    _db, circulation_data_source_name, autocreate=True
+                    _db, circulation_data_source_name, autocreate=True,
+                    offers_licenses=True
```

Comment:
In this case we know it offers licenses because the OPDS feed said so. It's the data_source_name that's passed into the class that could be the metadata wrangler. 

You could move up the comment right below this, and take out the below line that sets offers_licenses to true. It's probably safer to not change the licensing status of an existing data source, even if an OPDS feed told us it offers licenses.